src/components/SuperHeader/SuperHeader.jsx

```diff
@@ -42,4 +37,17 @@ const HelpLink = styled.a`
   }
 `;
 
+const Wrapper = styled.div`
+  font-size: 0.875rem;
+  color: ${COLORS.gray[300]};
+  background-color: ${COLORS.gray[900]};
+
+  height: 40px;
```

Comment:
Although matches design, it's better if we use `rem` since the header should scale with text size.
40 / 16 = 2.5 so `height: 2.5rem`